### DISCUSSION TITLE
bundler and theme are required while init a vuepress project through npm

### DISCUSSION THREAD

**Discussion by Jaylee89** [vuepress/core#862] [Q&A] [answered]
Follow this guide to init a vuepress project through `npm`, but get error `bundler and theme are required`, why?

```
"vuepress": "^2.0.0-beta.41"
```

https://v2.vuepress.vuejs.org/zh/guide/getting-started.html#%E6%89%8B%E5%8A%A8%E5%AE%89%E8%A3%85

**Comment by Mister-Hope** [comment]
~~You need to set bundler and theme now. @meteorlxy  Docs may need an update.~~

Edited: https://stackblitz.com/fork/vuepress-8ur7dq the demo is working for me. If you have any troubles, a same minimal reproduction repo is required.

**Comment by meteorlxy** [comment, marked as answer]
See #869
